ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -89,6 +89,7 @@ descriptor:
   dc: "Unidirectional flow of Electric Charge"
   dehumidification: "Process of removing moisture from air."
   detection : "Process of identifying conditions."
+  direction : "Direction in which particular entity is moving or is present "
```

Comment:
nit: please remove extra space before the colon and at the end of the description